### DISCUSSION TITLE
List of notifications.argoproj.io subscribe on

### DISCUSSION THREAD

**Discussion by tppalani** [argoproj/argo-cd#18438] [Q&A]
can you please some one help me what are the list of notification argopro will support i know only blow things which i have added under the argo application notification metadata value.

```
notifications.argoproj.io/subscribe.on-deployed.teams:  channel-name
notifications.argoproj.io/subscribe.on-health-degraded:  channel-name
notifications.argoproj.io/subscribe.on-sync-failed.teams:  channel-name
notifications.argoproj.io/subscribe.on-sync-running.teams:  channel-name
notifications.argoproj.io/subscribe.on-sync-succeeded.teams:  channel-name
```

**Comment by agaudreault** [comment]
Take a look at https://argo-cd.readthedocs.io/en/stable/user-guide/subscriptions/ and https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/triggers/. The notifications are configurable and depends on which triggers were defined in your instance of ArgoCD.
